load/UpdateDBPBibleFileTags.py

```diff
@@ -121,12 +121,11 @@ def process(self, hashId, inputFileset):
 				if tagValue == None:
 					insertRows.append((thumbnailFilename, bibleFileId, 'thumbnail', 0))
 				else:
-					updateRows.append((thumbnailFilename, bibleFileId, tag))
+					updateRows.append(("value", thumbnailFilename, tagValue, bibleFileId, tag))
 
 		tableName = "bible_file_tags"
 		attrNames = ("value",)
-		self.dbOut.insert(tableName, ("file_id", "tag", "admin_only"), attrNames, insertRows)
```

Comment:
This seems unrelated to language_translations. please remove. There is ticket DBP-499 to change updateCol when unrelated to language_translation

Reply:
@bradflood , It has been removed from this PR.